patching/src/main/java/org/jboss/as/patching/logging/PatchLogger.java

```diff
@@ -238,4 +238,7 @@
 
     @Message(id = 47, value = "Cannot copy files from %s to %s: %s")
     IOException cannotCopyFiles(String from, String to, String reason, @Cause Throwable cause);
+
+    @Message(id = 48, value = "Unable to remove some of the history files.")
+    String failedToRemoveHistoryFiles();
```

Comment:
This can return IllegalStateException, and then the callers can just do this:

throw PatchLogger.ROOT_LOGGER.failedToRemoveHistoryFiles();